_posts/2015-11-25-mouseless-programming.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+name: mouseless-programming
+title: 'Mouseless Programming'
+date: 2015-11-25 17:00:00 +00:00
+author: Tomaz Tekavec
+image:
+    src: "/assets/img/custom/blog/dilbert.jpg"
+    attribution:
+        text: Scott Adams, Still Pumped from Using the Mouse SC 
+        href: https://d1466nnw0ex81e.cloudfront.net/n_iv/600/1038055.jpg
```

Comment:
I'm not even sure we're allowed to reproduce it here without prior permission. The [dilbert.com terms](http://dilbert.com/terms) only allow redistribution of the collaborative content (which I understand as comments).

As far as I understand, fair use only applies if your article is talking about the artwork (eg a movie review can include movie stills and the movie poster). 

That being said, I have seen so many uses of Dilbert in conference slides...